### ISSUE TITLE
Facilitate ability to re-use a notebook week-over-week

### ISSUE THREAD

**Issue by edhille** [GaloisInc/ASKE-E#53]
We should build out a notebook with a timeframe specified at the top that would allow the user to reuse the notebook later, only having to adjust the date at the top.

**Comment by abakst** [comment]
In other words - support a workflow that allows the user to easily run a set of measurements etc. on new data sets.

**Comment by edhille** [comment]
Yeah. One of the things we realized was that a common workflow is likely to be someone creating a notebook to evaluate a set of models and wanting to be able to easily reevaluate them over time to continually assess their skill. @m10f might have more context, but that's how I currently understand it.
